omod/src/main/java/org/openmrs/module/pihcore/page/controller/patient/EncounterListPageController.java

```diff
@@ -19,7 +19,7 @@
 
     public void get(PageModel model, UiUtils ui,
                       @InjectBeans PatientDomainWrapper patientDomainWrapper,
-                      @RequestParam(value = "patient") Patient patient,
```

Comment:
HFE-UI automatically applies the patient id as a "patientId" parameter on any return url it generates, so adapting this to work with that.  (Would have been better to call it "patient", but it is what it is... the underlying mapper can handle both ids and uuids).